### ISSUE TITLE
[Bug][A11y]: Action-Menu menu items doesnt read in screen-readers

### ISSUE THREAD

**Issue by Rajdeepc** [adobe/spectrum-web-components#4556]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Action-Menu, Picker

### Expected behavior

Screen readers should be able to read menu items.

### Actual behavior

Screen readers not able to read menu items.

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-menu--default
2. Open Voice Over
3. Tab to the Menu Button
4. Press Down Arrow and scroll through the menu items
5. Note that screen reader doesn't read the content of the menu items

**Comment by nikkimk** [comment]
So it's a cross root ARIA issue, which means we can't use `aria-active descendant` as the `sp-menu` is in `sp-action-menu`'s shadowDOM but the slotted `sp-menu-item` elements are lightDOM, so the ID references will not work cross root.
